cea/plots/demand/energy_balance.py

```diff
@@ -253,13 +253,10 @@ def calc_monthly_energy_balance(data_frame, normalize_value):
 
 def main():
     import cea.config
-    import cea.inputlocator
     import cea.plots.cache
     config = cea.config.Configuration()
-    locator = cea.inputlocator.InputLocator(config.scenario)
-    # cache = cea.plots.cache.PlotCache(config.project)
     cache = cea.plots.cache.NullPlotCache()
-    EnergyBalancePlot(config.project, {'building': locator.get_zone_building_names()[0],
+    EnergyBalancePlot(config.project, {'building': config.plots.building,
```

Comment:
@gabriel-happle This was actually me, not @daren-thomas (his user was signed in on the computer I made these edits from).

Changing this line allows you to run energy_balance.py with a selected building from the config file. Before, you could either run all plots for a building specified in the config file (even if you didn't need/want the other plots) or you could run the energy_balance plot for whatever the first building in the zone shapefile is. I think this change adds useful functionality.